### ISSUE TITLE
USER ISSUE: Connection timed out

### ISSUE THREAD

**Issue by DucDeBretagne** [StrangeLoopGames/EcoIssues#10820]
**Version:** 0.7.8.9 beta

**Steps to Reproduce:**

**Expected behavior:**
It should have logged me up on the server i wanted

**Actual behavior:**

I don't have a bad connection at all (31ms, 24Mbps down, 11Mbps up) and my firewall is set up accordingly but every time i want to log on a server, this error message shows up :
The following error message popped up:
```
<size=150%>Connexion échouée</size>

Connection timed out
```

**Do you have mods installed? Does issue happen when no mods are installed?:**
The following error message popped up:
```
<size=150%>Connexion échouée</size>

Connection timed out
```

**Comment by johnkslg** [comment]
Hopefully fixed with your networking changes coming.

**Comment by mirasrael** [comment]
Should be fixed in 8.0. Please re-open if you continue experience issues with connection.
